packages/eslint-config-airbnb-base/rules/imports.js

```diff
@@ -68,7 +68,17 @@ module.exports = {
     // https://github.com/benmosher/eslint-plugin-import/blob/master/docs/rules/no-extraneous-dependencies.md
     // paths are treated both as absolute paths, and relative to process.cwd()
     'import/no-extraneous-dependencies': ['error', {
-      devDependencies: ['spec/**', 'test/**', 'tests/**', '**/__tests__/**'],
+      devDependencies: [
+        'spec/**', 
+        'test/**', 
+        'tests/**', 
+        '**/__tests__/**',
+        '**/webpack.config.js',
```

Comment:
why would anyone have their webpack config not at the root? the leading "**/" should be removed - same for all the other config files.